src/helpers/article.js

```diff
@@ -0,0 +1,121 @@
+import Models from '../models/index';
+import { ApplicationError } from './errors';
+
+const uuidRegularExpression = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/;
+
+const {
+  Articles,
+  Categories,
+  ArticleCategories
+} = Models;
+
+/**
+ * Returns article array or undefined
+ *
+ * @function
+ *
+ * @param {string} title - title of the article to be sort
+ *
+ * @returns {Array} - array or undefined
+ */
+export const getArticles = async (title) => {
+  const article = await Articles.findAll({
+    where: {
+      title
+    }
+  });
+  return article;
+};
+
+/**
+ * find id in given table and returns article array or undefined
+ *
+ * @function
+ *
+ * @param {uuid} id - id to be sort
+ * @param {Object} table - table to querry
+ *
+ * @returns {Object} - array or undefined
+ */
+export const idPresent = async (id, table) => {
+  if (id) {
+    const Id = await table.findOne({
+      where: {
+        id
+      }
+    });
+    return Id;
+  }
+};
+
+/**
+ * validates given tags and throws error if any is invalid
+ *
```

Comment:
Can we put these model helpers in their respective model files?

Reply:
> Can we put these model helpers in their respective model files?

Please @Merkll, what are the reasons for this because I refactored my code yesterday and removed repeating code to helper functions so as to implement DRY and keep the controllers clean.
I believe keeping the functions in the helper folder is enough.

Reply:
That helps ensure reusability but I think the model should have those responsibilities

Reply:
Also, ensure your success and error response conforms to the convention

Reply:
Okay, I see your point.
I have added the necessary functions in the models folder